clients/fides-js/rollup.config.mjs

```diff
@@ -11,12 +11,12 @@ import commonjs from "@rollup/plugin-commonjs";
 
 const NAME = "fides";
 const IS_DEV = process.env.NODE_ENV === "development";
-const GZIP_SIZE_ERROR_KB = 30; // fail build if bundle size exceeds this
-const GZIP_SIZE_WARN_KB = 15; // log a warning if bundle size exceeds this
+const GZIP_SIZE_ERROR_KB = 55; // fail build if bundle size exceeds this
```

Comment:
I'm pretty uncomfortable with how fat this is getting! Notable sizes:

1. Current size (w/o any localization): 24kb
2. With all static strings, including (unused) TCF strings: 44kb
3. With only the GPC static strings, excluding TCF strings: 32kb

Reply:
Created a follow-up in PROD-1796